functionaltests/src/test/java/specs/response/HealthResponseFixture.java

```diff
@@ -0,0 +1,36 @@
+package specs.response;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.fluent.Request;
+import org.apache.http.entity.ContentType;
+import org.apache.http.impl.client.BasicResponseHandler;
+import org.concordion.api.MultiValueResult;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.json.JSONObject;
+import org.junit.runner.RunWith;
+
+/**
+ * Simple /health URL response checker
+ */
+@RunWith(ConcordionRunner.class)
+public class HealthResponseFixture extends BasePaymentFixture {
+
+    public String getUrl() throws Exception {
+        return getHealthUrl();
+    }
+
+    public MultiValueResult handleHealthResponse() throws Exception {
+        final HttpResponse httpResponse = Request.Get(getHealthUrl())
+                .connectTimeout(200)
```

Comment:
But this is only a test! Real `/health` source implementation has default Integration Service timeout (as i know, for Spark it is 120 sec)